src/tree-view/internal.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import clsx from 'clsx';
+
+import { warnOnce } from '@cloudscape-design/component-toolkit/internal';
+
+import { getBaseProps } from '../internal/base-component';
+import { fireNonCancelableEvent } from '../internal/events';
+import { InternalBaseComponentProps } from '../internal/hooks/use-base-component';
+import { TreeViewProps } from './interfaces';
+import InternalTreeItem from './tree-item';
+
+import styles from './styles.css.js';
+import testUtilStyles from './test-classes/styles.css.js';
+
+type InternalTreeViewProps<T> = TreeViewProps<T> & InternalBaseComponentProps;
+
+const InternalTreeView = <T,>({
+  expandedItems,
+  items = [],
```

Comment:
We should not assign items to an empty array here because items prop is required. If it fails in tests - we should modify the default component props setting instead.

Also, all property defaults if we need some must be set in the index file in order to be correctly parsed by the documenter.